### ISSUE TITLE
"Video is not supported" when network type set to Custom:br0 (macvlan)

### ISSUE THREAD

**Issue by HVR88** [Unlearned6688/unraid-templates#3]
Found an interesting issue..

When the container's network type is set to Bridge (using host's IP) everything seems to work correctly and video will play within the browser, as tested on various system and browser combos (MacOS and iOS).

But, when network type is set to Custom:br0 (macvlan on its own IP), initial setup, search and browsing work, however, video won''t ever play back in the browser and always displays the error "Video is not supported"

There is no difference whether the webUI and server are accessed with IP or FQDN. Bridge works, macvlan custom doesn't.

**Comment by Unlearned6688** [comment]
I haven't tried it with macvlan setup. My docker network is setup using ipvlan.

It *does* work for me using br0 ipvlan though. I'm a bit hesitate to start playing around too much with my docker network settings... so, I'll just take your word that macvlan might have problems.

I tried 5 different shows/movies. 4/5 played. The 1 that didn't was an older movie, 1960s era. So possibly the torrent cache is bad.

![ipvlan-setup](https://github.com/user-attachments/assets/3f0d7540-f16f-41af-91de-937cbe40d625)

I'll add a note in the read revision that using br0 might have issues vs bridged or custom network. most people just bridged anyway i'm pretty sure. or one of a few custom docker networks.

**Comment by HVR88** [comment]
So this stopped happening. I have no idea why, but then again, I also have no idea how Stremio stores all its settings. I cleared out browser and DNS caches, restarted server and later when I set the server URL again, it "just worked."

So if you're going to include any kind of note, maybe suggest once everything is set up on the docker/Unraid side, reset all browser cache/cookies/session data and restart before giving it a go.
